lib/resources/r4/immunization_create.yaml

```diff
@@ -0,0 +1,318 @@
+---
```

Comment:
Shouldnt have separate YAMLs based on the action. Please combine them

Reply:
Removed the separated yaml and merged all in to one yaml file.
https://github.com/cerner/fhir.cerner.com/pull/276/commits/075c87c022842809e7760dfd0cfc46328d9e8c1c